Dockerfile

```diff
@@ -1,24 +1,28 @@
-FROM ruby:2.6.6-alpine3.10
+FROM ruby:2.6.9-alpine3.15
 
-RUN apk update && apk add jq bash build-base nodejs python3 py3-pip
+RUN apk add --update --no-cache jq bash build-base nodejs python3 py3-pip imagemagick
 
-RUN gem install bundler
-RUN gem install rails
+RUN gem install bundler -v 2.1.4 \
+  && gem install rails -v '~>6.0.3' \
+  # Remove unneeded files (cached *.gem, *.o, *.c)
+  && rm -rf /usr/local/bundle/cache/*.gem \
+  && find /usr/local/bundle/gems/ -name "*.c" -delete \
+  && find /usr/local/bundle/gems/ -name "*.o" -delete
 
 LABEL "com.github.actions.name"="generate_ssm_document"
 LABEL "com.github.actions.description"="generates an ssm document"
 LABEL "com.github.actions.icon"="upload-cloud"
 LABEL "com.github.actions.color"="purple"
 
-LABEL version="0.1.0"
+LABEL version="0.2.0"
 LABEL repository="https://github.com/rewindio/github-action-generate-ssm-documents"
-LABEL homepage="https://www.rewind.io/"
+LABEL homepage="https://www.rewind.com/"
 LABEL maintainer="Harrison Hammond <harrison@rewind.io>"
 
 # https://github.com/aws/aws-cli/blob/master/CHANGELOG.rst
-ENV AWSCLI_VERSION='1.18.2'
+ENV AWSCLI_VERSION='1.22.55'
 
-RUN pip3 install --quiet --no-cache-dir awscli==${AWSCLI_VERSION}
+RUN pip3 install --quiet --no-cache-dir awscli==$AWSCLI_VERSION
```

Comment:
a python interpreter might not be needed at all if awscli v2 is installed?

Reply:
looks like awscliv2 installation needs more packages for it to compile.
also I can't spare the time to bump the major and have to test all the cli commands being used :^|
Will leave it as is.